.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,31 @@
+  issues:
+    types: [ opened, unlabeled ]
+  pull_request:
+    types: [ opened, unlabeled ]
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  label-check:
+    runs-on: ubuntu-latest
+    outputs:
+      status: ${{ steps.label-check.outputs.status }}
```

actionlint

property "label-check" is not defined in object type {}

(expression)